src/main/java/spoon/reflect/visitor/ImportCleaner.java

```diff
@@ -118,7 +118,7 @@ protected void handleTypeReference(CtTypeReference<?> reference, Context context
 		}
 	}
 
```

Comment:
If these inner classes become public, shouldn't them be documented?

Reply:
done!